.mockery.yaml

```diff
@@ -0,0 +1,16 @@
+with-expecter: true
+filename: "{{.InterfaceName | snakecase}}.go"
+dir: "internal/mocks"
+outpkg: "mocks"
```

Comment:
This puts all mocks in the same directory by default right? Is this the current behavior or is it genned in the local directory + `/mocks` path?

Reply:
In this PR I kept all locations the same as before. The default behavior was crazy, and put things in something like "mocks/github.com/smartcontractkit/chainlink-ccip/internal/reader". That didn't even work because of the internal package. Something like this might be good: `dir: mocks/{{ replaceAll .InterfaceDirRelative "internal" "internal_" }}`.

Reply:
I went ahead and made this change, now there are no overrides to the default locations when mocks are generated.